spec/controllers/api/v1/photos_controller_spec.rb

```diff
@@ -3,20 +3,46 @@
 module Api
   module V1
     describe PhotosController do
+      render_views
+
       let(:user) { create(:user) }
-      let!(:cake) { create(:creation) }
+      let!(:cake) { create(:creation, user: user) }
 
-      describe "#index" do
-        before :each do
-          request.env['HTTP_AUTHORIZATION'] = ActionController::HttpAuthentication::Token.encode_credentials(user.authentication_token)
-          user.creations << cake
-        end
+      before :each do
+        request.env['HTTP_AUTHORIZATION'] = ActionController::HttpAuthentication::Token.encode_credentials(user.authentication_token)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.